src/Release/Entity/PublicRelease.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace Clearbooks\Labs\Release\Entity;
```

Comment:
This should not be in the Entity namespace, you should put this in the ```Release\UseCase\GetPublicRelease``` namespace